cumulusci/cli/utils.py

```diff
@@ -0,0 +1,28 @@
+from collections import defaultdict
+
+
+def get_available_tasks(runtime):
+    return (
+        runtime.project_config.list_tasks()
+        if runtime.project_config is not None
+        else runtime.universal_config.list_tasks()
+    )
+
+
+def get_available_flows(runtime):
+    return (
+        runtime.project_config.list_flows()
+        if runtime.project_config is not None
+        else runtime.universal_config.list_flows()
+    )
```

Comment:
get_available_tasks and get_available_flows look like they would make sense as methods of the Runtime class, rather than standalone functions.